bulk_handlers_test.go

```diff
@@ -93,6 +93,16 @@ func TestCreateUserWithResourceOwnershipApplicationType(t *testing.T) {
 		t.Errorf("expected userid is %s instead of %s", testUserId, users[0].UserID)
 	}
 
+	var sources []m.Source
+	err = dao.DB.Model(&m.Source{}).Where("name = ?", nameSource).Find(&sources).Error
+	if err != nil {
+		t.Error(err)
+	}
+
+	if *(sources[0].UserID) != users[0].Id {
```

Comment:
I know that this change was in previous PR, but I think we can simplify `users[0].Id` only to `user.Id` when we save only one user from db:
```
	var user m.User
	err = dao.DB.Model(&m.User{}).Where("user_id = ?", testUserId).Find(&user).Error
	if err != nil {
		t.Error(err)
	}
```

Reply:
Changed 👍